test/markdownlint-test.js

```diff
@@ -1059,6 +1097,20 @@ tape("configBadChildFile", (test) => {
     });
 });
 
+tape("configBadChildPackage", (test) => {
+  test.plan(4);
+  markdownlint.readConfig("./test/config/config-badchildpackage.json",
+    function callback(err, result) {
+      test.ok(err, "Did not get an error for bad child file.");
+      test.ok(err instanceof Error, "Error not instance of Error.");
+      // @ts-ignore
+      test.equal(err.code, "ENOENT",
+        "Error code for bad child file not ENOENT.");
+      test.ok(!result, "Got result for bad child file.");
+      test.end();
```

Comment:
"file" -> "package"

Reply:
For lines 1104/1108/1109

Reply:
The problem is that this error comes from `fs.readFile` / `fs.readFileSync` in `readConfig`/`readConfigSync`. At that point we don't know if the resolution was done via `path.resolve` or `require.resolve`. Besides, if a config is like `@my-company/my-package/my-file.json` it may be that the package is resolved but the file in the package is non-existent.

Reply:
Agreed. But this test is named "...package", so I thought its error messages should say "package" instead of "file".